{"name": "test_form_demoqa", "status": "broken", "statusDetails": {"message": "ValueError: There is no such driver by url https://chromedriver.storage.googleapis.com/LATEST_RELEASE_118.0.5993", "trace": "def test_form_demoqa():\n        registration_page = RegistrationPage()\n        student = users.student\n    \n>       registration_page.open()\n\ntest_form_demoqa.py:9: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n..\\qa_guru_python_8_12\\pages\\registration_page.py:14: in open\n    browser.open('/automation-practice-form')\n..\\venv\\Lib\\site-packages\\selene\\browser.py:90: in open\n    return browser.open(absolute_or_relative_url)\n..\\venv\\Lib\\site-packages\\selene\\support\\shared\\browser.py:78: in open\n    self.config.get_or_create_driver().get(url)\n..\\venv\\Lib\\site-packages\\selene\\support\\shared\\config.py:281: in get_or_create_driver\n    return self._source.get_or_create()\n..\\venv\\Lib\\site-packages\\selene\\support\\shared\\config.py:129: in get_or_create\n    return self.create()\n..\\venv\\Lib\\site-packages\\selene\\support\\shared\\config.py:110: in create\n    self._stored = self._set_driver()\n..\\venv\\Lib\\site-packages\\selene\\support\\shared\\config.py:192: in <lambda>\n    lambda: self._set_chrome_or_firefox_from_webdriver_manager()\n..\\venv\\Lib\\site-packages\\selene\\support\\shared\\config.py:259: in _set_chrome_or_firefox_from_webdriver_manager\n    return {'chrome': get_chrome, 'firefox': get_firefox}.get(\n..\\venv\\Lib\\site-packages\\selene\\support\\shared\\config.py:243: in get_chrome\n    ).install()\n..\\venv\\Lib\\site-packages\\webdriver_manager\\chrome.py:37: in install\n    driver_path = self._get_driver_path(self.driver)\n..\\venv\\Lib\\site-packages\\webdriver_manager\\core\\manager.py:29: in _get_driver_path\n    binary_path = self.driver_cache.find_driver(driver)\n..\\venv\\Lib\\site-packages\\webdriver_manager\\core\\driver_cache.py:95: in find_driver\n    driver_version = driver.get_version()\n..\\venv\\Lib\\site-packages\\webdriver_manager\\core\\driver.py:42: in get_version\n    self.get_latest_release_version()\n..\\venv\\Lib\\site-packages\\webdriver_manager\\drivers\\chrome.py:44: in get_latest_release_version\n    resp = self._http_client.get(url=latest_release_url)\n..\\venv\\Lib\\site-packages\\webdriver_manager\\core\\http.py:33: in get\n    self.validate_response(resp)\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nresp = <Response [404]>\n\n    @staticmethod\n    def validate_response(resp: requests.Response):\n        status_code = resp.status_code\n        if status_code == 404:\n>           raise ValueError(f\"There is no such driver by url {resp.url}\")\nE           ValueError: There is no such driver by url https://chromedriver.storage.googleapis.com/LATEST_RELEASE_118.0.5993\n\n..\\venv\\Lib\\site-packages\\webdriver_manager\\core\\http.py:15: ValueError"}, "attachments": [{"name": "log", "source": "c9353e76-77fb-424c-b08a-a5572e9be109-attachment.txt", "type": "text/plain"}, {"name": "stdout", "source": "89ef2efd-c72b-4dd0-87ed-a84bfb1dd4e5-attachment.txt", "type": "text/plain"}, {"name": "stderr", "source": "6d03d6ad-0bff-437f-8b90-dd9e5d8a7186-attachment.txt", "type": "text/plain"}], "start": 1698728295464, "stop": 1698728297557, "uuid": "3d5dac56-a438-42cd-899a-8e096beeadd8", "historyId": "32dc684e215ecd8cfd24e159890f25d1", "testCaseId": "772c3306c7afa36c4ba56c473af374ea", "fullName": "tests.test_form_demoqa#test_form_demoqa", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_form_demoqa"}, {"name": "host", "value": "DESKTOP-U2EORTL"}, {"name": "thread", "value": "5012-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_form_demoqa"}]}